core/templates/dev/head/exploration_editor/editor_extensions.html

```diff
@@ -0,0 +1 @@
+<link rel="import" href="/extensions/interactions/MultipleChoiceInput/MultipleChoiceInput.html"> 
```

Comment:
I thought you mentioned during our meeting that the ``link`` tag wasn't properly importing the HTML's Angular ``script`` sections?